ASFAchievementManager/AchievementHandler.cs

```diff
@@ -90,16 +96,18 @@ public sealed class AchievementHandler : ClientMsgHandler {
 							if (int.TryParse(Achievement.Name, out int bitNum)) {
 								if (uint.TryParse(stat.Name, out uint statNum)) {
 									bool isSet = false;
-									if ((Response.stats != null) && (Response.stats.Find(statElement => statElement.stat_id == statNum) != null)) {
-										isSet = (Response.stats.Find(statElement => statElement.stat_id == statNum)!.stat_value & ((uint) 1 << bitNum)) != 0;
+									uint stat_value = 0;
+									if (Response?.stats?.Find(statElement => statElement.stat_id == statNum) != null) {
+										stat_value = Response.stats.Find(statElement => statElement.stat_id == statNum)!.stat_value;
+										isSet = (stat_value & ((uint) 1 << bitNum)) != 0;
 									};
 
 									bool restricted = Achievement.Children.Find(Child => Child.Name == "permission") != null;
 
 									string? dependancyName = (Achievement.Children.Find(Child => Child.Name == "progress") == null) ? "" : Achievement.Children.Find(Child => Child.Name == "progress")?.Children?.Find(Child => Child.Name == "value")?.Children?.Find(Child => Child.Name == "operand1")?.Value;
 
 									uint.TryParse((Achievement.Children.Find(Child => Child.Name == "progress") == null) ? "0" : Achievement.Children.Find(Child => Child.Name == "progress")!.Children.Find(Child => Child.Name == "max_val")?.Value, out uint dependancyValue);
-									string lang = CultureInfo.CurrentUICulture.EnglishName.ToLower();
+									string lang = CultureInfo.CurrentUICulture.Name.ToLower();
```

Comment:
Weird - I originally had issues when using `EnglishName` throwing `System.MissingMethodException` back in Novemeber which I had assumed was related to this change for ASF 5 - https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Plugins#native-dependencies

However reverting this back to `EnglishName` appears to work now, so either I was incorrect in my original assesment (likely) or recent ASF updates have now resolved this _(tested against 5.0.2.4)_.

Reply:
Well, let's say sometimes it's good to be a friend of developer https://github.com/JustArchiNET/ArchiSteamFarm/commit/198db60c5110404695a9a746018cfcb3a5a9fc53 😁 
But even apart from that, it won't work with `Name`, and `MissingMethodException` is not a reason to discard functionality. We can always use it only with `generic` ASF version to avoid `MissingMethodException` anyway.

Reply:
Agreed, I think this was something I meant to go back to to resolve and then forgot about when I first made these changes last year. Well spotted 😄